src/sdk/bucketspace_fs.js

```diff
@@ -175,6 +176,21 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
         }
     }
 
+    /**
+     * stat_bucket will return the stat output on bucket config file
+     * @param {string} bucket_name
+     * @returns {Promise<nb.NativeFSStats>}
+     */
+    async stat_bucket(bucket_name) {
+        if (config.ALLOW_BUCKET_NS_CACHE_STAT) {
+            try {
+                const bucket_config_path = this.config_fs.get_bucket_path_by_name(bucket_name);
+                return nb_native().fs.stat(this.fs_context, bucket_config_path);
```

Comment:
we should prefer interactions with config files inside config_fs, can you move it please? 
```
const bucket_config_path = this.config_fs.get_bucket_path_by_name(bucket_name);
return nb_native().fs.stat(this.fs_context, bucket_config_path);
```